package/run.sh

```diff
@@ -20,7 +20,32 @@ else
     FORCE_RESTART=false
 fi
 
-env "INSTALL_K3S_FORCE_RESTART=${FORCE_RESTART}" "INSTALL_K3S_SKIP_DOWNLOAD=true" "INSTALL_K3S_SKIP_SELINUX_RPM=true" "INSTALL_K3S_SELINUX_WARN=true" installer.sh $@
+
+# work around for https://github.com/k3s-io/k3s/issues/2306
+# depends on journald being available on first boot, so this may not work on all providers (e.g. Digital Ocean)
+DELAY=5
+while true
+do
+    if ! env "INSTALL_K3S_FORCE_RESTART=${FORCE_RESTART}" "INSTALL_K3S_SKIP_DOWNLOAD=true" "INSTALL_K3S_SKIP_SELINUX_RPM=true" "INSTALL_K3S_SELINUX_WARN=true" installer.sh $@; then
```

Comment:
Does this work despite `set -e` at the top of the file? Also can we do `while ! ` instead?

Reply:
Yeah I can change it. This works despite `set -e`, there's a section in [the docs](https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html) that covers behavior in while loops and if statements


> Exit immediately if a pipeline ... returns a non-zero status. The shell does not exit if the command that fails is part of the command list immediately following a while or until keyword, part of the test in an if statement, ...  or if the command’s return status is being inverted with ! ....